src/main/java/net/praqma/tracey/cli/Tracey.java

```diff
@@ -29,14 +23,24 @@ public static String readFileToString(String path) throws IOException {
 
     public static void main(String[] args) throws Exception {
         ArgumentParser parser = ArgumentParsers.newArgumentParser("tracey")
-        .description("Send message with Tracey. Using RabbitMQ\n\nRun with arguments"
-                + " 'say -h' for help in sending messages\nRun with arguments 'listen -h' for help receiving messages");
-
+        .description("Send message with Tracey. Using RabbitMQ\n\nRun with arguments" +
+                " 'say -h' for help in sending messages\nRun with arguments 'listen -h' for help receiving messages\n" +
+                "Please notice command line options will override values provided through the configuration file\n" +
+                "If no arguments will be provided, a connection will be configured with default values:\n" +
```

Comment:
Could you please add info about environment variables support for userName and Password? see example here https://github.com/Praqma/tracey-broker/blob/master/README.md

Please use values from RabbitMQDefaults instead of hardcoded default values